docs/components/receive.md

```diff
@@ -126,6 +126,9 @@ Flags:
                                  Alternative to 'receive.hashrings-file' flag
                                  (lower priority). Content of file that contains
                                  the hashring configuration.
+      --receive.hashrings-algorithm=hashmod
```

Comment:
Kind of sad our tooling does not put those values in help. Do you mean explicitly adding those options for people to know how to use it?